src/css/styles.css

```diff
@@ -201,4 +201,178 @@ section {
     .store img {
 
     }
+}
+
+ul {
```

Comment:
style dla wszystkich ul już są w body

Reply:
A to ul akurat nie ja ustawiałam :> Ale usunę ;)